### ISSUE TITLE
Installation problems: "Custom element doesn't exist: numberbox-card."

### ISSUE THREAD

**Issue by JohannesKlug** [htmltiger/numberbox-card#83]
I'm trying to install this card following the instructions. My Home Assistant runs in Docker, and this is the first custom card I'm trying.

I placed the javascript file in <config>/www/, and added the lovelace configuration lines. I then restarted the Docker container.
This part seems to have worked, as I can now retrieve the JS file from my server using curl:
curl http://192.168.x.y:8123/local/numberbox-card.js

However, when trying to add a custom card to my dashboard, I get this error:
Custom element doesn't exist: numberbox-card.
type: custom:numberbox-card

![image](https://user-images.githubusercontent.com/208371/231863478-800c8c6b-dbd0-4824-99b1-83b8e460ecb3.png)

Can somebody help me troubleshoot this? Many thanks!

**Comment by nepozs** [comment]
Instead old way adding lovelace extensions you could try moderm way from GUI (I don't know if is it working with Docker/core  installations, but nothing to loose):
![lovelace_modern_way_01_2023-04-13_21-54](https://user-images.githubusercontent.com/56661567/231869475-de69eb84-2c83-4790-8205-90341ea28be5.png)
![lovelace_modern_way_02_2023-04-13_21-54](https://user-images.githubusercontent.com/56661567/231869491-d3c8935d-03d8-44bf-a7bf-e30663f5cda3.png)
![lovelace_modern_way_03_2023-04-13_21-54](https://user-images.githubusercontent.com/56661567/231869514-fdea125a-3590-4493-bff6-1dbbdaf25625.png)

**Comment by JohannesKlug** [comment]
Many thanks for the tip, @nepozs!
I only have the first three menu items available, "Manage resources" is missing. Do you happen to know how I could enable it?

**Comment by nepozs** [comment]
Maybe it is disabled by entry in `configuration.yaml` (just delete it or comment out)
```
lovelace:
  resources:
[…] 
    - url: /local/numberbox-card.js?v=1
      type: module
# I don't know if any other resources are defined, so remove all in section above
# if you have more of them, they should be readded by GUI
```
and restart HA

One tip more:
after big changes in lovelace (HA Dashboard) you should delete browser cache (usually in browser settings)

**Comment by Martin12350** [comment]
> ![lovelace_modern_way_01_2023-04-13_21-54](https://user-images.githubusercontent.com/56661567/231869475-de69eb84-2c83-4790-8205-90341ea28be5.png) ![lovelace_modern_way_02_2023-04-13_21-54](https://user-images.githubusercontent.com/56661567/231869491-d3c8935d-03d8-44bf-a7bf-e30663f5cda3.png) ![lovelace_modern_way_03_2023-04-13_21-54](https://user-images.githubusercontent.com/56661567/231869514-fdea125a-3590-4493-bff6-1dbbdaf25625.png)

Please pin it to the installation guide (if done locally), I do not wanna admit how embarrassingly long it took me, until I found this. :D

**Comment by nepozs** [comment]
@Martin12350 
Manual installation is not recommended anymore, it is much better to use HACS for this purpose (e.g. in case of update required).

**Comment by Martin12350** [comment]
@nepozs I am just getting started to play with custom components on HA core, and HACS is not installed by default. 
Watching what everything is necessary to get started with HACS, I wanted something fast, which was copy the js file and include the resource. But I was trying to add it in the configuration.yaml and no matter what I did, it was ignored. Until found out this and worked like a charm. Anyway, sooner or later I will check HACS :)

Also card_mod and custom styles requires additional package if I understand it correctly, also would be nice to notice for beginners ;)
But that one I found pretty quickly by googling :) Did not test it yet.
